packages/amplify-provider-awscloudformation/src/amplify-service-manager.ts

```diff
@@ -17,7 +17,7 @@ export async function init(amplifyServiceParams) {
 
   let amplifyAppId;
   let verifiedStackName = stackName;
-  let deploymentBucketName = `${stackName}-deployment`;
+  let deploymentBucketName = `${stackName}-deploy`;
```

Comment:
I'm not sure this is safe. I.e. there might be plenty of logic that rely on deployment bucket name, and studio might be reading from it too.

Reply:
🤯 

I wish there're some test to confirm that. If not, we'll have to keep `deployment` and think about other ways to keep the bucket name short

Reply:
Unfortunately this might require digging to confirm :/

Reply:
revert